datadog/dogshell/common.py

```diff
@@ -58,8 +58,8 @@ def load(self, config_file, api_key, app_key):
                     report_errors({'errors': ['%s has no [Connection] section' % config_file]})
             else:
                 try:
-                    response = ''
-                    while response.strip().lower() not in ['y', 'n']:
+                    response = None
+                    while response is None or response.strip().lower() not in ['', 'y', 'n']:
                         response = get_input('%s does not exist. Would you like to'
                                              ' create it? [Y/n] ' % config_file)
                         if response.strip().lower() in ['', 'y', 'yes']:
```

Comment:
```suggestion
                        if response.strip().lower() in ['', 'y']:
```

Reply:
Hi @zippolyte,

Indeed, I didn't saw this one ! I applied the additional fix !